app/src/main/kotlin/no/nav/omsorgsdager/vedtak/infotrygd/InfotrygdInnvilgetVedtakService.kt

```diff
@@ -39,9 +40,13 @@ internal class InfotrygdInnvilgetVedtakService(
             )
         )}
 
+        // TODO: har vi noe fra infotrygd å slå sammen?
```

Comment:
Årskvantum henter nå alene-om fra rammemeldinger som spleiser det i infotrygd me det som er lagret der;

https://github.com/navikt/omsorgspenger-rammemeldinger/blob/ada1e8a98a397609e4f832eda7a782b65d9b9d53/src/main/kotlin/no/nav/omsorgspenger/aleneom/apis/SpleisetAleneOmOmsorgenService.kt#L21

Dette blir jo et tredje sted å lagre alene-om-omsorgen på.
Hvilke tanker er gjort når man huker av at man er alene om omsorgen for et barn f.eks. ifbm korona/ordinære overføringer? Per nå lagres det som at personen er alene om omsorgen for det barnet, skal ikke dette gjøres lengre?

Reply:
i utbetalingssøknad tenker du på?  Det vil fases ut

Reply:
Usikker på utbetaling, men hvert fall for overføring/koronaoverføring. Om søkeren huker av at de er alene om omsorgen for et eller fler barn så lagrer vi det slik at det hentes opp også ved en utbetalingssak

Reply:
du tenker på det de oppgir her?
https://www.nav.no/familie/sykdom-i-familien/soknad/deling-omsorgsdager/melding/om-barna

tror den spm kan fases ut, må avsjekke med Ina

Reply:
Ja stemmer. Da burde vi i tilfelle slutte å lagre det der, for så å flytte over det vi har lagret til denne tjenesten hit + flytte over denne logikken av spleising med hva som ligger i Infotrygd

Reply:
👍